packages/common/package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@eth-optimism/hardhat-ovm": "^0.2.2",
     "@nomiclabs/hardhat-ethers": "^2.0.2",
-    "@nomiclabs/hardhat-etherscan": "^2.1.4",
+    "@nomiclabs/hardhat-etherscan": "^2.1.6",
```

Comment:
Out of curiosity, what did this fix?

Reply:
in theory, this gives us optimism/arbitrum verification. in practice, this only works on their mainnet